File: News_Service_JDBC/src/main/java/com/dunice/projectNews/models/News.java
package com.dunice.projectNews.models;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.JoinTable;
import jakarta.persistence.ManyToMany;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.hibernate.annotations.Fetch;
import org.hibernate.annotations.FetchMode;
import java.util.Set;

@Entity
@Table(name = "news", schema = "news_schema")
@Getter
@Setter
@NoArgsConstructor
public class News {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "news_id", nullable = false)
    private Long newsId;

    @Column(name = "description", nullable = false)
    private String description;

    @Column(name = "image", nullable = false)
    private String image;

    @Column(name = "title", nullable = false)
    private String title;

    @ManyToMany
    @JoinTable(name = "news_tags",
            joinColumns = @JoinColumn(name = "news_id"),
            inverseJoinColumns = @JoinColumn(name = "tag_id"))
    private Set<Tag> tags;

    @ManyToOne
    @Fetch(FetchMode.JOIN)
    @JoinColumn(name = "users_id")
    private User author;
}

File: News_Service_Kotlin/src/main/kotlin/com/dunice/advancedServiceKotlin/models/News.kt
import com.dunice.advancedServiceKotlin.models.Tag
import com.dunice.advancedServiceKotlin.models.User
import jakarta.persistence.*

@Entity
@Table(name = "news", schema = "news_schema")
 class News(

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "news_id", nullable = false)
    var newsId: Long=0L,

    @Column(name = "description", nullable = false)
    var description: String="",

    @Column(name = "image", nullable = false)
    var image: String="",

    @Column(name = "title", nullable = false)
    var title: String="",

    @ManyToMany
    @JoinTable(
        name = "news_tags",
        joinColumns = [JoinColumn(name = "news_id")],
        inverseJoinColumns = [JoinColumn(name = "tag_id")]
    )
    var tags: Set<Tag> = HashSet(),

    @ManyToOne
    @JoinColumn(name = "users_id")
    var author: User,
)
